pkg/cli/tools/binary_tools.go

```diff
@@ -193,3 +172,16 @@ func getFilename(base string) (string, error) {
 		return "", fmt.Errorf("unsupported platform %s/%s", runtime.GOOS, runtime.GOARCH)
 	}
 }
+
+func getFileExtension() (string, error) {
```

Comment:
nit: these are filenames not file extensions. A file extension would be like `.exe`

Reply:
Updated